client_test.go

```diff
@@ -207,13 +207,7 @@ func TestHandleErrorResp(t *testing.T) {
 <hr><center>nginx</center>
 </body>
 </html>`)),
-			expected: `error, status code: 413, status: , body: <html>
-<head><title>413 Request Entity Too Large</title></head>
-<body>
-<center><h1>413 Request Entity Too Large</h1></center>
-<hr><center>nginx</center>
-</body>
-</html>`,
+			expected: `error, status code: 413, status: , message: invalid character '<' looking for beginning of value`,
```

Comment:
This expected error text is a bit misleading. Code 413 indeed points out to "Content Too Large" (generic HTTP error), but the text afterward applies to json de-serialization.

I agree that this approach is better in terms of preserving the code, but in this case we actually don't preserve data

Reply:
What if Error() for RequestError (https://github.com/sashabaranov/go-openai/blob/22cf414b4be89a38eb702b13c27be1de6bcef74d/error.go#L106C1-L108C2) is updated to return body also like:
fmt.Sprintf("error, status code: %d, status: %s, message: %s, body: %s", e.HTTPStatusCode, e.HTTPStatus, e.Err, e.Body)

Should I make this change?

Reply:
@sashabaranov I have updated this PR with the above mentioned change, now in error response the original error data is also present. Can you please review it?